mongogrant/server.py

```diff
@@ -17,7 +17,9 @@
 from mongogrant.config import ConfigError, Config
 
 
-path = os.path.join(os.getenv("HOME"), '.mongogrant.server.json')
+path = os.path.join(
+    os.path.expanduser("~") or os.path.expanduser("~"), ".mongogrant.server.json"
```

Comment:
```suggestion
    os.path.join(os.getenv("HOME") or os.path.expanduser("~"), ".mongogrant.server.json"
```

Reply:
I'm glad @tschaume's careful review caught this......
Thanks @jan-jenssen!